src/fu-util-common.c

```diff
@@ -537,6 +537,10 @@ fu_util_setup_networking (GError **error)
 		return NULL;
 	}
 
+	/* relax the SSL checks for broken corporate proxies */
+	if (g_getenv ("DISABLE_SSL_STRICT") != NULL)
```

Comment:
Given we've started adopting `FWUPD_` prefixes for fwupd specific behaviors, what do you think about adding that to the start of this instead?

Reply:
I _was_ planning to use the same env-var for gnome-software and GFU too...

Reply:
Ah okay.  One more thought then - how about making this command line option that causes the environment variable to export?

Reply:
I thought that this might be exported in `.bashrc` like HTTP_PROXY as well.

Reply:
Yeah I think that could work then if you pair `--disable-ssl-strict` as a command line option that only causes `DISABLE_SSL_STRICT` to export and outputs something like this:

```
g_print ("Ignoring SSL strict checks, to do this automatically in the future export DISABLE_SSL_STRICT in your environment");
```

Reply:
Done.